fairlearn/preprocessing/_information_filter.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) Microsoft Corporation and contributors.
+# Licensed under the MIT License.
+
+import numpy as np
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.utils import check_array
+from sklearn.utils.validation import check_is_fitted
+
+
+class InformationFilter(BaseEstimator, TransformerMixin):
```

Comment:
I am worried about our terminology here. _Filter_ is used frequently in signal processing and there might be some risk of confusion, e.g., https://en.wikipedia.org/wiki/Kalman_filter#Information_filter . Can we do something more descriptive? How about `CorrelationRemover` ? Other ideas?

Reply:
That can work.